### ISSUE TITLE
Disable future dates not working properly

### ISSUE THREAD

**Issue by kakadiadarpan** [airbnb/react-dates#629]
I'm trying to disable dates before today and dates after 120 number of days from today, like below:
```
**this.props.isOutsideRange**:
const mDay = moment(day);
    const cDate = moment();
    const dayDiff = mDay.diff(cDate, "days");

    if (dayDiff !== 0 && mDay.diff(cDate) < 0) {
      return true;
    }
    if (dayDiff > 120) {
      console.log(mDay);
    }
    return dayDiff > 120;
```
Usage of **DayPickerSingleDateController**:
```
<DayPickerSingleDateController
          date={ this.props.date }
          numberOfMonths={ 5 }
          orientation={ "verticalScrollable" }
          onDateChange={ this.props.onDateSelect }
          isOutsideRange={ this.props.isOutsideRange }
          hideKeyboardShortcutsPanel={ false }
        />
```

Now when I click on the navigation button to see more months, I see that all the dates that follow are selectable which should not be the case.

![screen shot 2017-07-18 at 10 09 41 pm](https://user-images.githubusercontent.com/10805593/28329041-03f54282-6c06-11e7-8682-c8acc8b31199.png)
![screen shot 2017-07-18 at 10 10 11 pm](https://user-images.githubusercontent.com/10805593/28329046-0a2b47c8-6c06-11e7-98fb-e75c37fc7567.png)
My start date is 18th July 2017, and 120 days from today is 16th November 2017. So all the dates after that should be disabled, but as you can see in the screenshot they are still enabled.

**Comment by majapw** [comment]
@kakadiadarpan Yes, this is a bug with the `VERTICAL_SCROLLABLE` calendar. 😱 I will try to get around to fixing it this weekend.

**Comment by daifee** [comment]
@majapw you forgot the bug ^_^

**Comment by dougmcmillan** [comment]
Seems to still be a bug in 2019, is there a workaround? Is VERTICAL_SCROLLABLE not supported? 

#898 and #1353 are related
